src/emulator/config.cpp

```diff
@@ -182,12 +182,16 @@ InitResult init(Config &cfg, int argc, char **argv) {
         po::options_description config_desc("Configuration");
         config_desc.add_options()
             ("archive-log,A", po::bool_switch(&cfg.archive_log), "Makes a duplicate of the log file with TITLE_ID and Game ID as title")
+            ("background-alpha,a", po::value(&cfg.background_alpha), "Background Alpha")
```

Comment:
I don't think people will specifiy these stuffs in command line, also they doesn't have debug value. Delete those changes (except log_uniforms).

Reply:
oki, i have delete